data/json/furniture_and_terrain/terrain-walls.json

```diff
@@ -7,7 +7,10 @@
       { "item": "rock", "count": [ 0, 2 ] },
       { "item": "wood_panel", "count": [ 0, 1 ] },
       { "item": "nail", "charges": [ 2, 8 ] },
-      { "item": "splinter", "count": [ 1, 5 ] }
+      { "item": "splinter", "count": [ 1, 5 ] },
+      { "item": "pipe", "count": [ 0, 2 ], "prob": 10 },
+      { "item": "cu_pipe", "count": [ 0, 2 ], "prob": 10 },
```

Comment:
Now that it's copper wire, you should be using charges instead of count.
This currently will spawn between 0 and 400 copper wire.